README.md

```diff
@@ -51,9 +494,9 @@ public class Stock
     // Class code here
 }
 ```
-### Supported operations
+### Supported Operations
```

Comment:
Should we choose between "Supported operations" and "SupportedOperations" as the section title? See my overall review comment.

Reply:
Thinking about it, I almost want to go with "Hydra Operation" in order to be really clear. I'm not sure if casing would be enough to suggest what is what to the uninitiated.

Reply:
Or maybe "Operation on resources and the Hydra Operation"